src/main/java/com/epam/lab/paymentsystem/utility/AddTestData.java

```diff
@@ -0,0 +1,137 @@
+
+package com.epam.lab.paymentsystem.utility;
+
+import com.epam.lab.paymentsystem.dao.ConnectionPool;
+import javax.naming.NamingException;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.LinkedList;
+import java.util.List;
+
+class AddTestData {
+
+    public static Connection connection;
```

Comment:
I think, you need `final` here, don't you?

Reply:
If I use the "final", I will need to immediately, when declaring a variable, handle the exception